src/cbaselib.c

```diff
@@ -492,21 +510,34 @@ int cosmoB_sChar(CState *state, int nargs, CValue *args) {
     return 1;
 }
 
+int cosmoB_sLen(CState *state, int nargs, CValue *args) {
+    if (nargs < 1) {
+        cosmoV_error(state, "string.len() expected 1 argument, got %d", nargs);
+        return 0;
+    }
+
```

Comment:
use IS_STRING() and check if the argument passed was a string, look at cosmoB_sByte for a reference on what that should look like

Reply:
Oops, added.